cmd/message_input.go

```diff
@@ -5,32 +5,48 @@ import (
 	"os"
 	"os/exec"
 	"strings"
+	"regexp"
+	"time"
+	"slices"
+	"fmt"
 
+	"github.com/sahilm/fuzzy"
 	"github.com/atotto/clipboard"
 	"github.com/ayn2op/discordo/internal/config"
 	"github.com/ayn2op/discordo/internal/consts"
 	"github.com/ayn2op/discordo/internal/ui"
+	"github.com/ayn2op/discordo/internal/cache"
 	"github.com/diamondburned/arikawa/v3/api"
 	"github.com/diamondburned/arikawa/v3/discord"
+	"github.com/diamondburned/arikawa/v3/state"
 	"github.com/diamondburned/arikawa/v3/utils/json/option"
+	"github.com/diamondburned/ningen/v3/discordmd"
 	"github.com/gdamore/tcell/v2"
+	"github.com/yuin/goldmark/ast"
 	"github.com/rivo/tview"
 )
 
 const tmpFilePattern = consts.Name + "_*.md"
+var mentionRegex *regexp.Regexp = regexp.MustCompile("@[a-zA-Z0-9._]+")
```

Comment:
Specifying the type is optional.
```suggestion
var mentionRegex = regexp.MustCompile("@[a-zA-Z0-9._]+")
```